### ISSUE TITLE
WARNING: The compass/css3/shared module has been deprecated

### ISSUE THREAD

**Issue by yuvilio** [at-import/toolkit#57]
Got this warning:

``` bash
WARNING: The compass/css3/shared module has been deprecated.
You can silence this warning by importing compass/css3/deprecated-support instead.
Please be aware that module will be removed in the next release.
         on line 1 of /home/yuvilio/.rvm/gems/ruby-2.0.0-p247@ruby247/gems/compass-core-1.0.0.alpha.16/stylesheets/compass/css3/\
_shared.scss
         from line 1 of /home/yuvilio/.rvm/gems/ruby-2.0.0-p247@ruby247/gems/toolkit-1.3.8/stylesheets/toolkit/_box-sizing.scss
         from line 4 of /home/yuvilio/.rvm/gems/ruby-2.0.0-p247@ruby247/gems/toolkit-1.3.8/stylesheets/toolkit/_border-box.scss
         from line 9 of /home/yuvilio/.rvm/gems/ruby-2.0.0-p247@ruby247/gems/toolkit-1.3.8/stylesheets/_toolkit.scss
         from line 1 of /home/yuvilio/ws/apps/laravel/lab/slider/public/sasslab/scss/styles.scss
```

Tested with this stack:

```
$ compass --version
Compass 1.0.0.alpha.17
Copyright (c) 2008-2014 Chris Eppstein
Released under the MIT License.
Compass is charityware.
Please make a tax deductable donation for a worthy cause: http://umdf.org/compass
$ sass --version
Sass 3.3.0.rc.2 (Maptastic Maple)
$ ruby --version
ruby 2.0.0p247 (2013-06-27 revision 41674) [x86_64-linux]
$
```

**Comment by Snugug** [comment]
Yup. Look at the prerelease (2.x.x) where this has been fixed, but some of the Compass dependent modules have been deprecated.

**Comment by yuvilio** [comment]
Oh great. I can install that via local build until it's available as a prerelease or main release.  Thanks @Snugug !

**Comment by Snugug** [comment]
It's currently available as `2.0.0.alpha.6`
